src/components/Search/UserList.vue

```diff
@@ -1,19 +1,20 @@
 <script lang="ts" setup>
 import UserListItem from './UserListItem.vue'
-import { User } from '/@/lib/apis'
+import apis from '/@/lib/apis'
 
 interface Props {
-  members: User[]
+  search: string
```

Comment:
あわせてこちらの名称も変更をお願いします
```suggestion
  query: string
```